src/diffusers/schedulers/scheduling_ddim_flax.py

```diff
@@ -131,19 +127,24 @@ def __init__(
             raise NotImplementedError(f"{beta_schedule} does is not implemented for {self.__class__}")
 
         self.alphas = 1.0 - self.betas
-        self.alphas_cumprod = jnp.cumprod(self.alphas, axis=0)
+
+        # HACK for now - clean up later (PVP)
+        self._alphas_cumprod = jnp.cumprod(self.alphas, axis=0)
```

Comment:
that's a hack for now - in the future IMO we should move all this logic to `create_state` so that the scheduler is fully stateless

Reply:
Agreed.